tcti/tcti_device.c

```diff
@@ -76,21 +70,12 @@ TSS2_RC LocalTpmSendTpmCommand(
                                    command_size,
                                    &offset,
                                    &cnt);
-    if (tcti_intel->status.debugMsgEnabled == 1) {
-        TCTI_LOG (tctiContext, rmPrefix, "");
-        TCTI_LOG (tctiContext,
-                  rmPrefix,
-                  "Cmd sent: %s\n",
-                  strTpmCommandCode (commandCode));
-        DEBUG_PRINT_BUFFER (rmPrefix, command_buffer, cnt);
-    }
+    
+    LOGBLOB_DEBUG(command_buffer, cnt, "Cmd sent: %s; Buffer:", strTpmCommandCode (commandCode));
```

Comment:
The strTpmCommandCode function has been removed.
Should this print the buffer also?

Reply:
strTpmCommandCode was removed...